detectors/node/opentelemetry-resource-detector-azure/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@opentelemetry/resource-detector-azure",
+  "version": "0.1.0",
+  "description": "OpenTelemetry SDK resource detector for Azure",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "compile": "tsc -p .",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version && lerna run version:update --scope @opentelemetry/resource-detector-azure --include-dependencies",
+    "prewatch": "npm run precompile",
+    "prepare": "npm run compile",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "watch": "tsc -w"
+  },
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@opentelemetry/api": "^1.0.0",
+    "@opentelemetry/contrib-test-utils": "^0.34.1",
+    "@types/mocha": "8.2.3",
+    "@types/node": "18.6.5",
+    "@types/sinon": "10.0.18",
+    "mocha": "7.2.0",
+    "nock": "13.3.3",
+    "nyc": "15.1.0",
+    "rimraf": "5.0.5",
+    "sinon": "15.2.0",
+    "ts-mocha": "10.0.0",
+    "typescript": "4.4.4"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.0.0"
+  },
+  "dependencies": {
+    "@opentelemetry/core": "^1.0.0",
+    "@opentelemetry/resources": "^1.0.0",
```

```suggestion
    "@opentelemetry/resources": "^1.10.1",
```

The `DetectorSync` interface is available from `1.10.0` if I recall correctly. However I'd recommend `^1.10.1` as there was a bug in the `1.10.0` resource package